services/slam/builtin/cartographer_int_test.go

```diff
@@ -124,7 +124,7 @@ func TestCartographerIntegration(t *testing.T) {
 
 	// Remove maps so that testing in offline mode will run in mapping mode,
 	// as opposed to updating mode.
-	test.That(t, os.RemoveAll(name+"/map"), test.ShouldBeNil)
+	test.That(t, resetFolder(name+"/map"), test.ShouldBeNil)
```

Comment:
[opt] think its fine that you made this change, but I do not think it was needed, as os.RemoveAll also showed that the slam service can handle if the folder was missing from the directory

Reply:
I don't understand. All that `resetFolder` does is call `os.RemoveAll`. The function already existed, which is why I figured we should use it. Do you mean that `resetFolder` implies through its name that the slam service can't handle if the folder is missing from the directory?